pymobiledevice3/cli/developer.py

```diff
@@ -271,6 +277,32 @@ def screenshot(service_provider: LockdownClient, out):
         out.write(Screenshot(dvt).get_screenshot())
 
 
+def app_lookup(install_service: InstallationProxyService, bundle_id: str):
+    # TODO: install_service.lookup is not working for RSD，use browse instead
```

Comment:
rebase on master. i added a `bundle_identifiers` keyword for that which also should be much faster